betatest/admin.py

```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8 -*-
+#
+# OpenCraft -- tools to aid developing and hosting free software projects
+# Copyright (C) 2015 OpenCraft <xavier@opencraft.com>
```

Comment:
My suggestion would be to change the email address and remove the year in a separate PR.  The year is bound to get outdated anyway.